src/tests/views/widgets/pill.test.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import * as sinon from 'sinon';
+import test from 'ava';
+import { shallow } from 'enzyme';
+import Pill from '../../../ts/views/widgets/pill';
+
+test('Pill renders as expected and onClick gets called', (t) => {
+  const onClickSpy = sinon.spy();
+  const wrapper = shallow(<Pill onClick={onClickSpy} type="red" removeable={true}>{'Test Pill'}</Pill>);
```

Comment:
what happens if i don't pass in an `onClick` function, and try to click it? Add a test for that case please!